data-raw/create_internal_data.R

```diff
@@ -18,14 +18,12 @@ sentiment_AFINN_df <- sentiments %>% dplyr::filter(lexicon == "AFINN")
 sentiment_AFINN <- sentiment_AFINN_df[["score"]]
 names(sentiment_AFINN) <- sentiment_AFINN_df[["word"]]
 
-exploratory_stopwords <- c("http", "https", "t.co")
+exploratory_stopwords <- c("http", "https", "t.co", "amp")
 default_stopwords = c(tm::stopwords("english"), exploratory_stopwords)
```

Comment:
If this is not used anywhere, please verify that and remove.

Reply:
Seached code and default_stopwords is not used. So removed it.